alacritty.yml

```diff
@@ -196,12 +196,13 @@
 # If `true`, bold text is drawn using the bright color variants.
 #draw_bold_text_with_bright_colors: false
 
-# Colors (Tomorrow Night)
+# Colors (base16 classic dark is used for normal colors and base16 default dark
+# for the bright ones)
 #colors:
   # Default colors
   #primary:
-  #  background: '#1d1f21'
-  #  foreground: '#c5c8c6'
+  #  background: '#181818'
+  #  foreground: '#d8d8d8'
```

Comment:
This is the original base16 classic white, right?

I'm not sure if I'm just used to my grey but this is **very** bright on top of `#181818`.

Reply:
The original has `#d0d0d0` against the `#151515`, and the default dark has `#d8d8d8` against the `#181818`, so if we use `#181818` I've picked the `#d8d8d8`. It's indeed bright, but it provides better contrast with white on blue and white on green, which are widely used.